cleanlab/datalab/datalab.py

```diff
@@ -188,6 +189,13 @@ def find_issues(
             To best detect label issues, provide this input obtained from the most accurate model you can produce.
 
             If provided, this must be a 2D array with shape (num_examples, K) where K is the number of classes in the dataset.
+            Must not be provided if `predictions` is provided.
```

Comment:
While resolving these things, I think it's better to look at pred_probs as the canonical/primary input for all tasks, unless specified otherwise (e.g. predictions for regression).

Reply:
Removed.